app/views/users/show.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="page-content">
+  <%= render 'layouts/header' %>
+  <%= render 'layouts/taskbar', origin: @user.name.upcase %>
+  <div class="container mb-4">
+    <div class="col-lg-6 invite-item d-flex justify-content-between px-3 py-3">
+      <div class="d-flex">
+        <span class="d-flex justify-content-center align-items-center profile-img"><i class="icon-user"></i></span>
+        <div class="d-flex flex-column ml-2">
+          <span><%= @user.name %></span>
+          <small class="status">@<%= @user.name.downcase %></small>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="content">
+    <div class="container show-user-profile pt-3">
+      <h4>HOSTED EVENTS</h4>
+      <div class="row post-list p-3">
+        <%- if @user.events.any? %>
+        <% @user.events.upcoming.each do |event| %>
```

Comment:
define three variables in the Users controller specifically on the show method that will hold own, upcoming, and past events.